.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,680 @@
+# Copyright (C) 2022 Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# SPDX-License-Identifier: curl
+
+name: spell
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - '**.md'
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  check:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: install pandoc
+      run: sudo apt-get install pandoc
+
+    - name: convert .3 man pages to markdown
+      run: find docs -name "*.3" -size +40c | sed 's/\.3//' | xargs -t -n1 -I OO pandoc OO.3 -o OO.md
+
+    - name: convert .1 man pages to markdown
+      run: find docs -name "*.1" -size +40c | sed 's/\.1//' | xargs -t -n1 -I OO pandoc OO.1 -o OO.md
+
+    - name: setup the custom wordlist
+      run: grep '^#W' .github/workflows/spellcheck.yml | sed 's/#W://' >  wordlist.txt
+
+    - name: accept all option names
+      run: find docs/libcurl/opts/ -name "*.3" | cut -d_ -f2 | sed 's/\.3//' | sort -u >> wordlist.txt
+
+    - name: Check Spelling
+      uses: UnicornGlobal/spellcheck-github-actions@master
+
+#W:ACTIVESOCKET
```

Comment:
Similar to #9520, I think these sorts of files should be mastered entirely separately from the CI definitions.

Reply:
And I have the same reasoning here: I don't think I can do comments in that file so copyright info can't be provided.

Reply:
Based on some of the words in this list, spaces are allowed. So, why not make the first word be

    #W:Copyright 2022 Daniel Stenberg, all rights reserved, etc.

Reply:
I would also like to use this word list to spell check Everything Curl. I've created my own list there but it would be nice to reuse this one. To make that easier, it would be nice if this word list were 1) in a separate file, and 2) a list of bare words, without the #W: prefix. That could be added in the workflow.